### PR TITLE
Improving our job offer

### PR DESCRIPTION
**Pull request by eiso**
@tsolakoua I have made some changes here that if you agree, I'd like to apply all offers.

### CODE REVIEW

File: talent/job-descriptions/applications.md [hunk 1]
```diff
@@ -1,6 +1,6 @@
 Engineer in the Applications team
```
Comment (tsolakoua) [inline talent/job-descriptions/applications.md]: I'd replace the "Engineer" with "Software Engineer", even though by default it's a software engineering role. I think it's more concise.

Comment (eiso) [reply talent/job-descriptions/applications.md]: +1, I changed it

**Comment by eiso** [comment]
@tsolakoua could you add the changes to the other offers (e.g. reordering of sections, example projects etc.) on this branch. 

If you like we can also on GitHub use sections instead, so the things that are repeated have their own file and are just linked. This might make it easier to maintain.

**Comment by tsolakoua** [comment]
@eiso Job descriptions added.

**Pull request merged**